app/views/layouts/application.html.haml

```diff
@@ -13,35 +13,36 @@
 
   %body
     #full-page-wrap
-      .navbar.navbar-fluid-top
-        .navbar-inner
-          .container-fluid
-            %a.btn.btn-navbar(data-target=".nav-collapse" data-toggle="collapse")
```

Comment:
There is some code that depends on this html class. It's only used by the bento theme. If this isn't needed anymore we can drop the js code. Otherwise we have to fix it?

https://github.com/openSUSE/travel-support-program/blob/master/app/themes/bento/assets/javascripts/bento/global-navigation.js#L30